src/parser/parser.cpp

```diff
@@ -368,10 +368,10 @@ bool Parser::isUnresolvedType(const std::string& name) {
 }
 
 std::vector<DatatypeType> Parser::mkMutualDatatypeTypes(
-    std::vector<Datatype>& datatypes, bool doOverload) {
+    std::vector<Datatype>& datatypes, bool doOverload, uint32_t flags) {
```

Comment:
This isn't clang-formatted (curly brackets go on their own lines).

Reply:
Done!